src/Tasks/ContentReviewEmails.php

```diff
@@ -24,19 +25,21 @@ class ContentReviewEmails extends BuildTask
 {
     private array $invalid_emails = [];
 
-    /**
-     * @param HTTPRequest $request
-     * @throws RuntimeException
-     */
-    public function run($request)
+    protected static string $commandName = 'content-review-emails';
```

Comment:
```suggestion
    protected static string $commandName = 'ContentReviewEmails';
```

Reply:
Discussion about this and other task command names is being held in the main issue.